mdbook-tera-backend/src/tera_renderer/custom_component.rs

```diff
@@ -0,0 +1,133 @@
+use serde::Deserialize;
+use std::cell::RefCell;
+use std::collections::{BTreeMap, HashMap};
+use std::path::{Path, PathBuf};
+
+use tera::Tera;
+
+use crate::{RendererError, Result};
+
+use super::RenderingContext;
+
+fn make_strip_prefix_function() -> impl tera::Function {
```

Comment:
You're "cheating" here by defining a helper — other users cannot do this, so we should avoid doing it too.

Reply:
They have a built in `trim_start_matches` (https://keats.github.io/tera/docs/) so we can remove this, we probably don't need extra methods